doc/developer/library.rst

```diff
@@ -15,5 +15,6 @@ Library Facilities (libfrr)
    hooks
    cli
    modules
+   lua
```

Comment:
should it go to developer, not to the user?

Reply:
this is not end user code so developer imo